company-keywords.el

```diff
@@ -142,10 +142,9 @@
       "tiny" "transfer" "transpose" "trim" "true" "type" "ubound" "unpack"
       "use" "value" "verify" "volatile" "wait" "where" "while" "with" "write"))
     (go-mode
-     ;; from https://golang.org/ref/spec#Keywords
```

Comment:
Could you bring back the comment? And add an extra explanation about the additional "keywords".

Reply:
👌